scripts/ipfs-pi-stream/uninstall

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+sudo systemctl stop process-stream
+sudo systemctl disable process-stream
+sudo rm -f /usr/bin/process-stream.sh
+sudo rm -f /etc/systemd/system/process-stream.service
+sudo systemctl daemon-reload
+
+# remove ffmpeg and supporting tools
+sudo apt-get -y remove ffmpeg lsof inotify-tools
```

Comment:
Is there any point it putting back the nginx directory permissions as they were before? I'm thinking no, but I just wanted to bring it up.